README.md

```diff
@@ -8,7 +8,7 @@ If you need help with anything CNCF + Summer of code related, you can file an is
 
 * Chris Aniszczyk ([@caniszczyk](https://github.com/caniszczyk)): https://twitter.com/cra
 * Ihor Dvoretskyi ([@idvoretskyi](https://github.com/idvoretskyi)): https://twitter.com/idvoretskyi
-* Cameron Zahedi ([@czahedi](https://github.com/czahedi))
+* Paris Pittman ([@parispittman](https://github.com/parispittman))
```

Comment:
@idvoretskyi I spoke with Paris. She has a lot on her plate and might not have enough bandwidth for this.

Can you add me here and add Paris as a shadow?

My github: https://github.com/nikhita
My twitter: https://twitter.com/TheNikhita

Paris's twitter: https://twitter.com/ParisInBmore

Reply:
I'd be fine with this